index.js

```diff
@@ -38,6 +38,7 @@ function fastifyWebsocket (fastify, opts, next) {
   const wss = new WebSocket.Server(wssOptions)
   fastify.decorate('websocketServer', wss)
 
+  websocketListenServer.removeAllListeners('upgrade')
```

Comment:
I am not sure if this is a correct approach.
Removing all the others listening is breaking changes and we remove all the ability for others to do anything on the `upgrade` event.
For example, they would like to log the event.

Instead, a proper fix should be inspect if `socket` already assigned.
Warn the developer they are mixing other `websocket` library and we are skipping the logic of that request.

Reply:
Yeah, I also feel like silently mutating the server is a bit unexpected. Is this is the only way to make it work? And if so, could we at least log a big deprecation warning or something that explains fastify-websocket doesn't play nice with any other libraries?

Reply:
I'm going to work on an alternative approach.